src/model.ts

```diff
@@ -59,6 +59,12 @@ export interface ICreateJobModel extends PartialJSONObject {
   scheduleWeekDay?: string;
 }
 
+export const InitialScheduleOptions = {
+  scheduleInterval: 'weekday',
+  schedule: '0 0 * * MON-FRI',
+  timezone: Intl.DateTimeFormat().resolvedOptions().timeZone
+};
+
```

Comment:
You don't need a separate object here. You should add `schedule: '0 0 * * MON-FRI'` into `emptyCreateJobModel()`. Then, whenever building a new create job model in the front end (i.e. in refill button or rerun job), you should pass the properties of this "empty" (which is actually confusingly non-empty) model via destructuring like so:

```
    const initialState: ICreateJobModel = {
      ...emptyCreateJobModel(),
      jobName: props.model.jobName,
      inputFile: props.model.inputFile,
      outputPath: props.model.outputPrefix ?? '',
      environment: props.model.environment,
      runtimeEnvironmentParameters: props.model.runtimeEnvironmentParameters,
      parameters: props.model.parameters,
      outputFormats: props.model.outputFormats,
    }
```

Renaming these model generators from `empty*` to `default*` or some other more semantically accurate name will be part of https://github.com/jupyter-server/jupyter-scheduler/issues/105. No need to do so here.